services/core-web/src/components/admin/MinespaceUserManagement.js

```diff
@@ -93,10 +127,14 @@ const mapStateToProps = (state) => ({
 const mapDispatchToProps = (dispatch) =>
   bindActionCreators(
     {
+      createMinespaceUser,
       fetchMineNameList,
       fetchMinespaceUsers,
       fetchMinespaceUserMines,
       deleteMinespaceUser,
```

Comment:
When clicking on the Delete icon for an existing minespace users record it displays the confirm text "Are you sure you want to delete undefined?", this is a known issue in Prod but I believe the reference just needs to be updated, it currently is "record.email" when I believe it should be "record.email_or_username". This isn't required for this ticket but would be a nice quick win if you are able.